odxtools/parameterinfo.py

```diff
@@ -58,7 +58,7 @@ def parameter_info(param_list: Iterable[Union[Parameter, EndOfPduField]]) -> str
         if isinstance(cm, TexttableCompuMethod):
             result += f": enum; choices:\n"
             for scale in cm.internal_to_phys:
-                result += f"  '{scale.compu_const}'\n"
+                result += f"  '{str(scale.compu_const)}'\n"
```

Comment:
Why?

Reply:
because mypy starts to complain about the change in behaviour for byte strings a few versions back. we also could use `f"{scale.compu_const!r}"` (what do you prefer?)

Reply:
I agree, but we don't know what type `scale.compu_const` actually is (it can be the representation of any atomic physical value, i.e.,  `int`, `str`, `bytes` or `float`), so we need to use something that works for all of these...

Reply:
(since this is a TEXTTABLE, the actual type must most likely be `str`, but `mypy` can't know that...)